easybuild/easyconfigs/a/ASAGI/ASAGI-1.0-foss-2023a.eb

```diff
@@ -0,0 +1,63 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jonney Huang <jonney.huang@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'CMakeMake'
+
+name = 'ASAGI'
+local_commit_id = 'f672507'
+version = '1.0'
+
+homepage = 'https://github.com/TUM-I5/ASAGI'
+description = """ a pArallel Server for Adaptive GeoInformation.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/TUM-I5/',
+        'repo_name': 'ASAGI',
+        'commit': local_commit_id,
+        'recursive': True,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+patches = [
+    ('ASAGI-CMakeList.patch', 0),
+    ('ASAGI-level.h.patch', 0),
```

Comment:
Patch files should be renamed to reflect the ASAGI version they're intended for, like `ASAGI-1.0_CMakeList.patch`